DirectEmployers/MyJobs
# PD-3460 analytics duplicate xhr requests on load

## What's this PR do?
Removes the redundant API calls on initial load of Analytics

## Where should the reviewer start?
1. `api.js`
2. `page-loading-actions.js`

## How should this be manually tested?
1. Open Dev Tools -> Network
2. Navigate to Analytics
3. There should only be 1 call to the `/available-reports` endpoint (formerly there were 3).
